### DISCUSSION TITLE
Why the inference always fallback to CPU?

### DISCUSSION THREAD

**Discussion by magic282** [open-compass/opencompass#663] [Q&A] [answered]
Hi all,

Thanks for the great easy-to-use tool.  I am facing an issue that the inference will fallback to CPU and not using GPU. Here is an screenshot:
![image](https://github.com/open-compass/opencompass/assets/2285145/69c3e790-c63d-4f63-a730-d662bc12dd1a)

My machine has 8*80GA100, and my command is:

```bash
HF_DATASETS_OFFLINE=1 TRANSFORMERS_OFFLINE=1 python run.py --datasets ceval_ppl --hf-path /workspace/data/open/baichuan/Baichuan2-7B-Base  --tokenizer-path /workspace/data/open/baichuan/Baichuan2-7B-Base  --tokenizer-kwargs padding_side='left' truncation='left' trust_remote_code=True  --model-kwargs device_map='auto' trust_remote_code=True   --max-seq-len 2048  --max-out-len 100  --batch-size 1  --no-batch-padding --num-gpus 1 --max-partition-size 512
```
Any help or hint is appreciated.

**Comment by yingfhu** [comment, marked as answer]
Your infer tasks were on GPUs already, 
![image](https://github.com/open-compass/opencompass/assets/42952108/b4b71724-f0e7-4a88-94ec-af5dbc0fa566)

The cpu tasks are for evalution which does not involves any GPU

**Reply by magic282** [reply]
I see. Thank you for the clarification.
